### PR TITLE
(SIMP-3335) Updated rsync to rsync_skeleton

### PR DESCRIPTION
**Pull request by nick-markowski**
SIMP-3335 #comment done in core

### CODE REVIEW

File: Puppetfile.stable [hunk 1]
```diff
@@ -4,8 +4,8 @@ mod 'simp-doc',
   :git => 'https://github.com/simp/simp-doc',
   :ref => '79c18f732f6c73b8bff21809c639771a892722f2'
 
-mod 'simp-rsync',
-  :git => 'https://github.com/simp/simp-rsync',
+mod 'simp-rsync'
```
Comment (jeefberkey) [inline Puppetfile.stable]: The point of the ticket was to rename the module in the `mod` line. There are two modules named `simp-rsync`and r10k complains. This does not address that issue.

Comment (jeefberkey) [inline Puppetfile.stable]: I don't have a problem with the name of the Git repo.

Comment (trevor-vaughan) [inline Puppetfile.stable]: @jeefberkey Oh, that's easy:

`mod 'simp_core-rsync'`

**Comment by trevor-vaughan** [comment]
I'd like to put this on hold until we hear back from https://github.com/puppetlabs/r10k/issues/759

Relates to https://github.com/simp/rubygem-simp-rake-helpers/pull/85

**Comment by trevor-vaughan** [comment]
We did this a different way in the new structure.

**Pull request closed without merging**